Blueprints/BlueprintDefinitions/vs2022/AnnotationsFramework/template/src/BlueprintBaseName.1/BlueprintBaseName.1.csproj

```diff
@@ -15,7 +15,7 @@
     <PackageReference Include="Amazon.Lambda.Core" Version="2.1.0" />
     <PackageReference Include="Amazon.Lambda.APIGatewayEvents" Version="2.6.0" />
     <PackageReference Include="Amazon.Lambda.Serialization.SystemTextJson" Version="2.3.1" />
-    <PackageReference Include="Amazon.Lambda.Annotations" Version="0.13.2" />
+    <PackageReference Include="Amazon.Lambda.Annotations" Version="1.0.0" />
   </ItemGroup>
   <ItemGroup>
     <FrameworkReference Include="Microsoft.AspNetCore.App" />
```

Comment:
We recently had a GitHub issue discussing confusion about this `FrameworkReference`. Can you add the following comment to make it clear why we are doing this.

```
  <!-- 
  The FrameworkReference is used reduce the deployment bundle size by not having to include 
  dependencies like Microsoft.Extensions.DependencyInjection. The Microsoft.AspNetCore.App
  which is available in the Managed .NET Lambda runtime already includes those assemblies.
  -->
  <ItemGroup>
    <FrameworkReference Include="Microsoft.AspNetCore.App" />
  </ItemGroup>
```

Reply:
Added